.github/workflows/ci.yaml

```diff
@@ -22,30 +22,56 @@ jobs:
         command: lint
         config: .github/ct.yaml
 
-  install-chart:
-    name: install-chart
+  build-image-install-chart:
     runs-on: ubuntu-latest
     needs:
-    - lint-chart
-    strategy:
-      matrix:
-        k8s:
-        - v1.17.5
-        - v1.19.1
+      - lint-chart
```

Comment:
optional - enable v1.17.[5|12] install test

Reply:
for this kind version, only v1.17.11 is available